src/pcapi/routes/serialization/venues_serialize.py

```diff
@@ -114,6 +97,35 @@ class EditVenueBodyModel(BaseModel):
     _dehumanize_venue_label_id = dehumanize_field("venueLabelId")
     _dehumanize_venue_type_id = dehumanize_field("venueTypeId")
 
+
+class VenueListItemSerializer(BaseModel):
```

Comment:
```suggestion
class VenueListItemResponseModel(BaseModel):
```

Ce n'est pas un serializer
Si je reprends ce qui a été fait pour `offers` avec les sous-modèle, on a pris le nommage que je t'ai mis en suggestion